src/pages/app.react.jsx

```diff
@@ -47,13 +53,13 @@ export default React.createClass({
       <div style={styles.mainDiv}>
         <div style={styles.mainView}>
           <Header/>
-          <Grommet.Box direction="row" align="center" justify="between" tag="aside">
-          <Grommet.Box pad='medium' style={styles.socialColumn}>
+          <Grommet.Box direction="row" align="start" justify="center" tag="aside">
+          <Grommet.Box pad='medium'>
             <Social/>
           </Grommet.Box>
-          <Grommet.Box colorIndex='grey-2' pad='large'>
-            {this.props.children}
-          </Grommet.Box>
+          <div style={styles.MainContainer}>
+            <MainContainer contentComponent={this.props.children} />
```

Comment:
it will be more elegant to just use it like this `<MainContainer>{this.props.children}</MainContainer>`